account-model/src/account/mod.rs

```diff
@@ -0,0 +1,56 @@
+use std::convert::TryFrom;
+
+#[derive(Debug, Eq, PartialEq)]
+pub enum Classification {
+    PERSONAL,
+    BUSINESS,
+}
+
+impl Classification {
+    pub fn value(&self) -> &'static str {
+        match *self {
+            Classification::PERSONAL => "Personal",
+            Classification::BUSINESS => "Business",
+        }
+    }
+}
+
+impl TryFrom<&str> for Classification {
+    type Error = String;
+
+    fn try_from(input: &str) -> Result<Self, Self::Error> {
+        match input {
+            "Personal" => Ok(Classification::PERSONAL),
+            "Business" => Ok(Classification::BUSINESS),
+            _ => Err(format!("{} is not classification variant", input))
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
to jest dziwne ze nie moge przenieść tego moda do osobnego pliku w `/tests` - nie widzi mi `Classification` ani `crate::account::Classification` ani `super::Classification` ale pewnie moze bedize to gdzies opisane przy `mod`-ach